scripts/deploy.sh

```diff
@@ -55,14 +62,22 @@ elif [[ ${TRAVIS_BRANCH} =~ ^release\-[0-9]+\.[0-9]+$ ]]; then
     # deploy release candidates in ???
     GCS_PROJECT="alpha"
     GCR_PROJECT="development-223016"
+    RELEASE_BUCKET="aether-releases"
 
 else
 
     DOCKER_VERSION="alpha"
     GCR_VERSION=${TRAVIS_COMMIT}
     GCS_PROJECT="alpha"
     GCR_PROJECT="development-223016"
-
+    RELEASE_BUCKET="aether-releases-dev"
```

Comment:
I'm not sure if you need the `export` option. I think I tried without it and the `push-app-version` didn't get it :thinking: